### ISSUE TITLE
Games slow psvita pemu

### ISSUE THREAD

**Issue by Vecinagamer** [Cpasjuste/pemu#133]
Games in general are slow examples.
1943 lacks fps about 5 fps,, 1944 the same,,operation wolf slow,,,golden axe slow,,ghost goblins slow, still lacks 6fps,,
Gauntlet 1 and 2 slow, hang on slow, super hang on slow.
The Arkanoid now works perfectly, the stock, thanks, we have been asking for it a lot

Games in general are slow

**Comment by Revendor** [comment]
Yes, i see too this problem. To many roms it's really slow on the Vita.
For example you can compare the game Fire Shark on the pFBA emulator and pFBNeo. pFBNeo it's slowest. Why? Shaders filters is ON? Check it, please.  
Thank you.

**Comment by Vecinagamer** [comment]
Thanks for your answer, but I can't disable the shaders in off in the emulator impossible

**Comment by renatotec** [comment]
> Thanks for your answer, but I can't disable the shaders in off in the emulator impossible

editing the config.cfg manually i've managed to disable, but it was no use, still very slow.

**Comment by npaladin2000** [comment]
> > Thanks for your answer, but I can't disable the shaders in off in the emulator impossible
> 
> editing the config.cfg manually i've managed to disable, but it was no use, still very slow.

How?  Just set it to NONE? I want to try the same thing with pSNES

**Comment by renatotec** [comment]
Set it to 'OFF'

**Comment by renatotec** [comment]
> How? Just set it to NONE? I want to try the same thing with pSNES

Set it to 'OFF'

The old version was not saving states, but was running at full speed. I'll go back to the old version for now

**Comment by npaladin2000** [comment]
Ok, setting to OFF worked...but you're right, it didn't change anything.

**Comment by Cpasjuste** [comment]
Hi,

This problem is now solved and will be available in the next coming days.

**Comment by Vecinagamer** [comment]
@Cpasjuste And could you arrange to be able to play the games vertically on psvita, please, that would be fantastic, the controls are inverted and I love 1943 vertically, thanks boss

**Comment by Cpasjuste** [comment]
Hi,

I think this should be fixed in [version 6.0](https://github.com/Cpasjuste/pemu/releases/tag/v6.0), can you confirm this ?

**Comment by Vecinagamer** [comment]
If I insert the roms in the roms folder, it gives an error and does not run the program in psvita, if the roms folder is empty, it does run the program in psvita

**Comment by Manfred282** [comment]
Right, it "should" be solved but it isnt ...

MAME games are still unplayable, Wardner, Thunder Fox, Chase H.Q., etc. does run at under 30fps

**Comment by Cpasjuste** [comment]
Isn't those games running slow on previous version too ?

**Comment by renatotec** [comment]
> Isn't those games running slow on previous version too ?

sorry for the delay in responding. Version 6.0 is still slow, I'm gonna test de new 6.1 version and give feedback, i've been using version 4.3 without any problems or slowdowns.

**Comment by Manfred282** [comment]
I did a quick test with pEMU 4.4 ... Chase H.Q. over 30fps but smooth and playable, Wardner still a bit slow biut 47fps compared to the 26fps with version 6.0  Other games seem to be way faster too with version 4.4 ...

Maybe i should try version 4.3 too, just to compare ;)

**Comment by Cpasjuste** [comment]
Thanks for your time! 
I know for sure 6.x is faster than 5.2, I tested both and you get around 15-20 fps boost (for example in 1944 cps2). If it's not the case then there's something wrong with the auto-builds. 
It's true that I didn't compare to 4.x, I hope it's not some changes in fbneo sources who generate this framerate drop because It could be hard to solve. 
I will take some time to debug this next wee.

**Comment by Vecinagamer** [comment]
Good,, as our friend Renatotec and Manfred282 say,, the Pemu 4.4 version is the best version,, the games hardly have fps slowdown,,, examples gauntlet 1 and 2, 1943, 1943 kai, out run, shinobi, robocop, in 4.4 Pemu it goes perfect at 60 fps..
It even works great to play games like 1943, 1942 vertically, version 4.4 is great, too bad the stick doesn't work like in 6.1 for games like Arkanoid, operation wolf, out run, it would be the best version if it could be changed only this.

**Comment by Cpasjuste** [comment]
The goal is not to fix 4.4 but to fix 6.x :)
Will try to work on that next week if I find the time.

**Comment by Vecinagamer** [comment]
I'm sorry but I confused the version,, the version with which the games work well is 1.54 Pfba,, 1943, shinobi, outrun, gauntlet 1 and 2 all those that are slow in the Pemu 6.1 version,,, in Pfba 1.54 go perfect

**Comment by Cpasjuste** [comment]
Hi @Manfred282 (and other), could you try this version (unzip to get vpk) ? :
[pfbneo.zip](https://github.com/Cpasjuste/pemu/files/8661210/pfbneo.zip)
It seems a lot better. I also added a new option called "WAIT_RENDERING" which can be disable and allow a little more fps (but doesn't work with shaders).

**Comment by Cpasjuste** [comment]
Hi, should be fixed in [v6.4](https://github.com/Cpasjuste/pemu/releases/tag/v6.4) (currently building/uploading), will reopen the issue if it's not the case.
